docs/requirements.txt

```diff
@@ -1,4 +1,5 @@
 tqdm
 librosa<0.7.0
 numpy>=1.16
-six
\ No newline at end of file
+six
+dali-dataset>=1.0
```

Comment:
could you take this opportunity to alphabetize our requirements?

Reply:
yep